content/en/network_monitoring/devices/setup.md

```diff
@@ -59,7 +60,8 @@ To monitor individual devices:
 
     ```yaml
     init_config:
-      loader: core
+      loader: core                     # use core check implementation of SNMP integration. recommended
+      use_device_id_as_hostname: true  # recommended
```

Comment:
Shouldn't we explain a little bit what is the effect of this option?

Reply:
@gmarabout I thought about adding `use_device_id_as_hostname: true  # use device id as hostname, recommended`

but it's a bit redundant, since the config name is quite clear.